### ISSUE TITLE
language tag matching (Accept-Language) vs RFC4647 (Trac #113)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#113]
Matching of language tags is defined in RFC4647.

Should the definition of "Accept-Language" just delegate to that specification?

Migrated from https://trac.ietf.org/ticket/113
```json
{
    "status": "closed",
    "changetime": "2008-08-04T10:20:21",
    "_ts": "1217845221000000",
    "description": "Matching of language tags is defined in RFC4647.\n\nShould the definition of \"Accept-Language\" just delegate to that specification?",
    "reporter": "julian.reschke@gmx.de",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-04-17T11:15:42",
    "component": "p3-payload",
    "summary": "language tag matching (Accept-Language) vs RFC4647",
    "priority": "",
    "keywords": "",
    "milestone": "04",
    "owner": "julian.reschke@gmx.de",
    "type": "design",
    "severity": "Active WG Document"
}
```

**Comment by mnot** [comment]
Trac update at 20080729T17:27:50: 
* julian.reschke@gmx.de commented:

> See <http://lists.w3.org/Archives/Public/ietf-http-wg/2008JulSep/0167.html>.


* julian.reschke@gmx.de changed milestone from "unassigned" to "04"
* julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"

**Comment by mnot** [comment]
Trac update at 20080730T00:46:27: fielding@gbiv.com commented:

> It can delegate by specifying a particular matching algorithm
> defined by RFC4647.
>

**Comment by mnot** [comment]
Trac update at 20080801T19:00:41: 
* julian.reschke@gmx.de changed attachment from "" to "i113.diff"
* julian.reschke@gmx.de commented:

> Proposed patch.

**Comment by mnot** [comment]
Trac update at 20080802T06:38:48: 
* julian.reschke@gmx.de changed attachment from "" to "i113.2.diff"
* julian.reschke@gmx.de commented:

> Updated proposal.

**Comment by mnot** [comment]
Trac update at 20080804T10:20:21: 
* julian.reschke@gmx.de commented:

> Fixed in [303]:
> 
> Resolve #113: refer to RFC4647 for definition of language ranges and matching (closes #113).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"
